privacyidea/lib/tokens/remotetoken.py

```diff
@@ -218,34 +218,24 @@ def check_otp(self, otpval, counter=None, window=None, options=None):
         :rtype: int
         """
         otp_count = -1
-        otpval = otpval.encode("utf-8")
 
         remoteServer = self.get_tokeninfo("remote.server") or ""
-        remoteServer = remoteServer.encode("utf-8")
 
         # in preparation of the ability to relocate privacyidea urls,
         # we introduce the remote url path
         remotePath = self.get_tokeninfo("remote.path") or ""
```

Comment:
With this change, ``removePath`` is not stripped anymore. I'm not sure it's necessary?

Reply:
Oh, totally missed that. Will ad a test for remote.path as well.

Reply:
The background is, that the remote privacyIDEA server could run e.g. at https://remote.example.com/pi2 while the originial privacyIDEA server would run at https://local.exampl.com/

Reply:
Fixed in 6141b198

Reply:
:+1: